src/integrated/soil_canopy_model.jl

```diff
@@ -179,6 +181,7 @@ forward in time, including boundary conditions, source terms, and interaction
 terms.
 """
 function SoilCanopyModel{FT}(;
+    toml_dict::CP.AbstractTOMLDict,
```

Comment:
there is an equivalent method for LandModel (takes component types and args) - should that be updated too?

Reply:
I believe @kmdeck is talking about [this one](https://github.com/CliMA/ClimaLand.jl/blob/ae349142d91734cd925aed0ea6c5bfbb0e08c8bf/src/integrated/land.jl#L133-L144). We're planning to remove this type of constructor soon, but while we still have them it would be good for them to remain consistent with each other